src/alcaramba/Drawable/Market/Market.gd

```diff
@@ -11,8 +11,8 @@ onready var money_market = get_node("%MoneyMarket")
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
-	Global.stack_tiles.shuffle()
 	money_market.connect("money_card_selected", self, "_on_money_card_selected")
+	_on_EndTurnButton_pressed()
```

Comment:
kann nur hier kommentieren: endTurn should rescale all tiles to one (if turn is ended with selected tile, it stays big)

Reply:
Fixed